test/browser_tests/page_objects/page_activity-log.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+function ActivityLog() {
+  this.get = function() {
+    browser.get( '/activity-log/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+  this.mainTitle = element( by.css( '[data-qa-hook="main-title"]' ) );
+
+  this.mainSummary = element( by.css( '[data-qa-hook="main-summary"]' ) );
+
+  this.searchFilter = element( by.css( '.js-post-filter' ) );
```

Comment:
Make sense.  Fixed in https://github.com/cfpb/cfgov-refresh/commit/2270d9d4a713a80407b4ef991a279426b5fca8fc